docs/linuxcredstores.md

```diff
@@ -87,7 +88,37 @@ export GPG_TTY=$(tty)
 **Note:** Using `/dev/tty` does not appear to work here - you must use the real
 TTY device path, as returned by the `tty` utility.
 
-## 3. Plaintext files
+## 3. Git's built-in [credential cache](https://git-scm.com/docs/git-credential-cache)
+
+```shell
+export GCM_CREDENTIAL_STORE=credentialcache
```

Comment:
Don't forget to update the table of values in both `docs/environment.md` and `docs/configuration.md` for the new option!

Reply:
..and the new `_OPTIONS` command line args option too.

Reply:
D'oh, good catch! Thanks!